docs/UserGuide.md

```diff
@@ -632,7 +628,7 @@ Wraps a constraint. Is satisfied if, and only if, its inner constraint is _not_
 ```javascript
 { "anyOf": [
     { "field": "foo", "is": "null" },
```

Comment:
The documentaion for null in the user guide is not quite finished.

Reply:
Addressed